includes/User.php

```diff
@@ -2174,6 +2174,8 @@ public function setPassword( $str ) {
 		}
 
 		$this->setInternalPassword( $str );
+		if($clearTokens)
+			wfRunHooks( 'SetPassword', [ $this->getId() ] );
```

Comment:
Wrap the second line in `{ }` brackets and follow MediaWiki whitespace convention.